### PR TITLE
Add virtualized version of input combobox

### PR DESCRIPTION
**Pull request by DanielaBeCause**

### CODE REVIEW

File: src/InputCombobox/VirtualizedInputCombobox.tsx [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+  borderRadius: "var(--BU)",
+  cursor: "pointer",
+  userSelect: "none",
+  backgroundColor: p.multiple ? "transparent" : computeColor([p.colorBackground, 700]),
+
+  "&[data-selected='true']": {
+    backgroundColor: computeColor([p.colorBackgroundHover, 100]),
+  },
+
+  // border: !p.multiple && p.selected ? `dotted 1px ${computeColor([p.colorBackgroundHover, 200])}` : "none",
```
Comment (ralobecause) [inline src/InputCombobox/VirtualizedInputCombobox.tsx:46]: Can probably be removed(?)

File: src/InputCombobox/VirtualizedInputCombobox.tsx [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+      playwrightTestId,
+    } = props
+
+    const [open, setOpen] = useState(false)
+    const [search, setSearch] = useState("")
+
+    const [filteredItemIds, setFilteredItemIds] = useState<string[]>([])
+    const [selectedItemIds, setSelectedItemIds] = useState<string[]>([])
+
+    const [height, setHeight] = useState(1)
```
Comment (ralobecause) [inline src/InputCombobox/VirtualizedInputCombobox.tsx:145]: We can't default to LIST_HEIGHT? Or 0?

File: src/InputCombobox/VirtualizedInputCombobox.tsx [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+
+    const filterWithDebounce = useMemo(() => debounce(filterResults, 300), [filterResults])
+
+    const filteredItems = useMemo(() => {
+      const filteredItemIdsSet = new Set(filteredItemIds)
+
+      return Object.entries(items)
+        .filter(([id]) => filteredItemIdsSet.has(id))
+        .map(([, value]) => value)
+    }, [filteredItemIds, items])
```
Comment (ralobecause) [inline src/InputCombobox/VirtualizedInputCombobox.tsx:226]: Hmm trying to visualize the component from the code so might get this wrong; do we need to add search here as dependency? If search changes, filtered items changes, no?

File: src/InputCombobox/VirtualizedInputCombobox.tsx (hunk 2 above)
Comment (ralobecause) [reply src/InputCombobox/VirtualizedInputCombobox.tsx:145]: Or I suppose that's a max? Perhaps we could call it MAX_LIST_HEIGHT

**Review by ralobecause — APPROVED** [review]
Super cool, nicely done

File: src/InputCombobox/VirtualizedInputCombobox.tsx (hunk 3 above)
Comment (DanielaBeCause) [reply src/InputCombobox/VirtualizedInputCombobox.tsx:226]: `search` changes immediately, but `filteredItemIds` changes after a debounce, so having search here will result in more calls to `useMemo` but the result will not change until `filteredItemIds` will not change.

**Pull request merged**